crates/nu-command/tests/commands/glob.rs

```diff
@@ -40,6 +40,42 @@ fn nonempty_glob_lists_matching_paths() {
     })
 }
 
+#[test]
+fn empty_glob_list_matches_default() {
```

Comment:
Clever use of `glob` as a command to generate a `ListStream` (I don't think we have any command that truly guarantees that so could change if we bork something with the implementation). I think for self-documentation purposes of what you want to assert here I would be inclined to place this test under `default` and explicitly mention that is related to the behavior on list *streams*.